rosdep/base.yaml

```diff
@@ -1579,6 +1579,11 @@ libboost-program-options-dev:
   fedora: [boost-devel]
   openembedded: [boost@openembedded-core]
   ubuntu: [libboost-program-options-dev]
+libboost-python:
```

Comment:
I'm wondering if we should make this key `libboost-python2`; clearly on Alpine and Fedora, there's a separate package for Python 3.  But I don't know what our policy on python2 vs. python3 rosdep keys is, so I'm not sure.

Reply:
At least on ubuntu, both the python2 and the python3 version are supported when installing `libboost-python-dev`. There is no specialized package for a specific version. 

Afaik, the ROS policy is not to enforce a specific python version but instead work with whatever is set as `PYTHON_VERSION` in CMake. Therefore it makes sense to ensure that boost packages are installed for all possible values of `PYTHON_VERSION`.